### PR TITLE
Widget

### PR DESCRIPTION
**Pull request by jishnu7**
temporary widget code. later, we will be moving this to a separate module.

### CODE REVIEW

**Review by rohanhc — APPROVED** [review]

**Review by bijoshtj — APPROVED** [review]

**Review by suhailars — APPROVED** [review]

**Comment by rampr** [comment]
Should we keep it in a separate branch until we make it generic ?

File: tealeaf/widget/TodayViewController.h [hunk 1]
```diff
@@ -0,0 +1,13 @@
+//
+//  TodayViewController.h
+//  widget
+//
+//  Created by panda on 04/10/16.
```
Comment (rampr) [inline tealeaf/widget/TodayViewController.h]: who is panda ?

File: tealeaf/widget/TodayViewController.h [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+//  Copyright © 2016 Game Closure. All rights reserved.
```
Comment (rampr) [inline tealeaf/widget/TodayViewController.h]: why copyright GameClosure ?

File: tealeaf/widget/TodayViewController.m [hunk 3]
```diff
@@ -0,0 +1,78 @@
+//
+//  TodayViewController.m
+//  widget
+//
+//  Created by panda on 04/10/16.
+//  Copyright © 2016 Game Closure. All rights reserved.
```
Comment (rampr) [inline tealeaf/widget/TodayViewController.m]: same comments as above

Comment (jishnu7) [reply tealeaf/widget/TodayViewController.m]: xcode added it. forgot to remove.

**Comment by jishnu7** [comment]
@rampr changed target to hc-widget.

**Review by rampr — APPROVED** [review]

**Pull request merged**